### PR TITLE
Update example README.md

### LINKED ISSUE #55: example README seems to be missing `gsutil` command
When following the instructions in `example/README.md`, I encounter an error when running `terraform plan`:

```
Error: Failed to get existing workspaces: querying Cloud Storage failed: googleapi: Error 403: terraformsa@<GCP_PROJECT>.iam.gserviceaccount.com does not have storage.objects.list access to <GCP_PROJECT>-terraform-state., forbidden
```

This error disappears once the following comand has been run, and `terraform plan` generates a plan as expected:

```bash
gsutil iam ch serviceAccount:${TF_GCP_SA}:objectAdmin gs://${GCS_TF_BUCKET}
```

### PR DESCRIPTION
**Pull request by wwwil**
This PR updates the example `README.md` to make the commands easier to copy and paste using bash variables. It also adds use of the `gsutil iam ch` command, and closes #55.

Signed-off-by: wwwil <wwwil.squires@gmail.com>

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -1,32 +1,21 @@
 # Terraform Google GKE Cluster Example
 
-This example project gives the minimum Terraform setup required to use the `terraform-google-gke-cluster` module.
+This example project gives the minimum Terraform setup required to use the [`terraform-google-gke-cluster`](https://registry.terraform.io/modules/jetstack/gke-cluster/google/) module.
```
Comment (charlieegan3) [inline example/README.md]:
```suggestion
This example project gives the minimum Terraform setup required to use the [`terraform-google-gke-cluster`](https://registry.terraform.io/modules/jetstack/gke-cluster/google/) module implemented in the rest of this repo.
```

File: example/README.md [hunk 2]
```diff
@@ -42,28 +31,48 @@ gcloud services enable container.googleapis.com
 gcloud services enable iam.googleapis.com
 ```
 
+## Setup
```
Comment (charlieegan3) [inline example/README.md]: I wonder if it's necessary to explain the configuration of remote state - beyond suggesting that people do it?

Comment (wwwil) [reply example/README.md]: I initially read this as suggesting I should explain remote state more, and added some links. On rereading I'm not so sure haha!

Currently the example `README.md` is aimed at people who are quite new to GKE and Terraform as it runs though setting up `terraform.tfvars`, creating a service account, and creating a remote state GCS bucket. I have tried to keep it concise though and block commands together so you can just copy and paste them if you already know what's going on. I assume people who are familiar with Terraform and GKE will just look at the main `README.md`.

Comment (charlieegan3) [reply example/README.md]: Ahh sorry - I understand the confusion. My suggestion was the opposite - that we just assume local state or they manage their state in the way that suits them.

This keeps this page about the module.

Sorry for the confusion.

**Pull request merged**